### ISSUE TITLE
Low contrast for Linux icon

### ISSUE THREAD

**Issue by kra-mo** [localsend/localsend#978]
### Describe the bug

Related to #864

The contrast for the Linux icon on stock GNOME as well as most desktops with a dark theme is just too low to be legible.

Aside from it looking bad, this is an accessibility concern.

![Screenshot from 2023-12-03 15-25-59](https://github.com/localsend/localsend/assets/93832451/7fe06808-1bae-4097-9e44-bedfd31304b2)

If you don't control what background an icon is going to show up on, it is best practice to give it a light or dark background by default (depending on what the icon is like).

### To reproduce

Install the app

### Expected behavior

The icon has a light background

### Screenshots

![image](https://github.com/localsend/localsend/assets/93832451/4e5e311d-7f2c-43e4-bc7e-35a88ad1aaf6)

**Comment by sergd88** [comment]
I agree that the icon doesn't look very contrasty, and its color is somewhat dull, but using a background for it's a bad idea, as I've mentioned before. Not to mention that a dark background under the icon will also be difficult to read and look bad in a dark theme, while a light color will be similarly problematic in a light theme. If Windows or Linux distributions used a unified icon style, like macOS does, this problem wouldn't arise at all. Cinnamon in Linux Mint partially uses a unified form for icons, but only for system ones or those developed by their team.

From the screenshot you provided, it's clear that it reads normally, just like any other monocolor, non-bright icon on such a background. I'm attaching a screenshot for comparison with AirDrop (whose icon isn't much better readable) on dark/light backgrounds. Many applications use such monocolor icons, so it's a common issue. In Windows 10, they addressed this by placing a uniform background behind the icon, but it looked awful. In macOS, iOS, and Android, this solution looks good. In GNOME, the large full-screen app menu means big icons, and with a background, you'll see the icon well. But in the equally popular KDE, the app menu is more like Windows, and icon with background will look small and bad, regardless of the background shape. With a circular background, the icon looks like a soccer ball when it's small. Just like in Windows.

I use both Ubuntu with GNOME and Kubuntu with KDE, not as often as before, but still. I tried the icon with a background on them. But it's generally just as bad as on Windows, although on GNOME, it's better due to the full-screen menu compared to Windows or KDE as I mentioned earlier. Ultimately, for me, without a background is better.

Perhaps with the UI redesign planned for the future, the icon will change somewhat, it'll see. I don't know how global the changes will be. But then, with the redesign, this issue can be raised. At this stage, the icon looks fine; with a background, it definitely won't become better or more readable.

![1](https://github.com/localsend/localsend/assets/125032334/33ee3628-6eb6-419b-9c70-62b04df063bc)

**Comment by kra-mo** [comment]
> the icon will also be difficult to read and look bad in a dark theme, while a light color will be similarly problematic in a light theme.

And why exactly would you want to read the background? You don't. You want to read the actual icon, for which a background provides sufficient contrast.

> I'm attaching a screenshot for comparison with AirDrop

Which is always shown against a light background for this reason...

<img width="90" alt="image" src="https://github.com/localsend/localsend/assets/93832451/ef32f0a7-b767-432c-9411-df31983177f9">

> Many applications use such monocolor icons, so it's a common issue.

So? It is still an issue that should be addressed. 


All of this is literally in the Flathub guidelines.

> avoid single-color glyphs and the like since they can be very hard to see if the background color is similar

https://docs.flathub.org/docs/for-app-authors/appdata-guidelines/quality-guidelines/

This is kind of just common knowledge and I don't know why you're arguing against it.

**Comment by sergd88** [comment]
I won't waste time on all these explanations, as they've already been partially disclosed and clarified in my lengthy previous comment. Your phrases like "Which is always shown against a light background for this reason..." or "All of this is literally in the Flathub guidelines" directly convey that any further explanations to you're simply a futile waste of time.

So, in summary, it's easier to just bring it all down to one point. You've already proposed this change with the same solution, to which the app author responded: "Not everybody like the circular white background". And if the background, for example, won't be round, but rectangular, rounded, isn't important, the reaction will be the same. You brought up this topic again – great, you've the right to do so. I noted that, I don't like this option and explained why. So let's end it here.

**Comment by kra-mo** [comment]
Your "explanations" were a high school essay's worth of "I just don't like how it looks".

> Not everybody like the circular white background

Cool, nothing is going to be beloved by everyone.

I personally don't like the fact that the status quo explicitly goes against wildly recognized standards about contrast and the guidelines of the platform that the app is published on.

**Comment by bbb651** [comment]
I disagree that it's an accessibility concern, the Contrast app is designed to test contrast between text and backgrounds (not saying that it's necessarily not one either, but until at least one person actually finds it a problem it's not one).

I do think however that it makes the logo look worse aesthetically. This also applies to the iOS 18 dark icon variant:
![sent with LocalSend :)](https://github.com/user-attachments/assets/332c9c2b-1a6d-4992-ac22-37829c3c6c5d)

I think a really nice solution would be to use the existing lighter color already used within the app in dark mode:
![image](https://github.com/user-attachments/assets/610be402-a6e4-4271-800a-63947c3efc07)
For iOS this can be conditional for only the dark mode variant (actually to properly follow [the HIG](https://developer.apple.com/design/human-interface-guidelines/app-icons#iOS-iPadOS) it should be a gradient but I don't think it's that big a deal), but for linux there's no standard (yet?). I personally think the lighter variant can replace the icon entirely, the lighter icon on a white icon looks a lot better than the dark icon on a dark background, and on linux a dark background is the majority (GNOME always has a dark background for the shell even in light mode, on other desktops it'll depend on whether you have dark mode enabled), not sure about how this affects other platforms.
